modules/rest-api/pages/rbac.adoc

```diff
@@ -618,6 +619,130 @@ An attempt to add a user to one or more non-existent groups fails with `400 Bad
 
 The creation of an already existing group or local or external user succeeds with `200 OK`: the user is recreated with the newly specified role-assignments and group-memberships; and the group is recreated with the newly specified role-assignments and, optionally, a newly specified mapping.
 
+[backup-restore]
+== Backup and Restore Users and Groups
+
+Use the `/settings/rbac/backup` endpoint to back up and restore users and groups. 
+It is useful if you want to transfer or synchronize users between different database clusters. 
+
+=== Backup Users and Groups
+
+Use the `GET` method with the  `/settings/rbac/backup` endpoint to backup all users and groups in the database, including the Full Administrator account. 
```

Comment:
No, 

The fact that the format is opaque and can change from version to version doesn't mean that newer versions will not support old formats. 

Actually we should maintain backward compatibility for backups. At least for users and user groups there will be backward compatibility. I can't speak for other types of backups.

@hyunjuV 
Do we have any understanding currently for how long we should maintain backward compatibility for backup format?

Reply:
@ggray-cb 
We shouldn't say anything about future compatibility.   

We'll maintain backwards compatibility for backup formats for as long as possible.  This isn't something we should be documenting ahead of time (since we don't know ourselves and there's no reason to put ourselves into a corner).  When something changes, we'll document then.

cc @timofey-barmin

Reply:
OK, changed to say we return an opaque string, with a clarification that the format for this string can change in the future (just in case anyone looks at the output and sees it is currently JSON).